.vib/rabbitmq/goss/goss.yaml

```diff
@@ -0,0 +1,31 @@
+command:
+  rabbmitmqctl-version:
+    exec: rabbitmqctl version | grep --quiet $APP_VERSION
+    exit-status: 0
+    stderr: [ ]
+    timeout: 20000
+  rabbitmqctl-add-vhost:
+    exec: rabbitmqctl add_vhost test-host
+    exit-status: 0
+    stderr: [ ]
+    stdout:
+      - 'Adding vhost'
```

Comment:
Does the command above only output this line? I'm not sure if the command could start with `Adding vhost` but failing at the end. If the commands returns something like "vhost added", it would be great.

Reply:
Here is an example of a successful and erroneous command execution.

`I have no name!@rabbitmq-0:/$ rabbitmqctl add_vhost test-host1245
Adding vhost "test-host1245" ... `

`I have no name!@rabbitmq-0:/$ rabbitmqctl add_vhost t!#@$`

Error (argument validation): too many arguments.

Taking this into account, do you think it's acceptable to have it like this?

Reply:
You are right, the command itself only outputs the `Adding...` line and I can't find a more fail-proof option. This is perfectly fine.